MarkBind/markbind
# Shorter syntax for using Glyphicons

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Documentation update
• [X] Enhancement to an existing feature


Fixes #185 



**What is the rationale for this request?**
User would like to use shorter syntax for using "Glyphicons". 
Difference in syntax reflected in updated user guide.

**What changes did you make? (Give an overview)**
- Modified `Site.js` in `collectUserDefinedVariablesMap` to populate  glyph name mappings for each page's map
- Created a new function `_getGlyphiconsMap` to retreive mappings from asset file in lib
- Created new folder in lib, named  `asset` for storing the `glyphicons.csv` file
- Updated [UserGuide](https://deploy-preview-209--markbind.netlify.com/userguide/contentauthoring#use-glyphicons)

**Provide some example code that this change will affect:**

```js
Old:
<span class="glyphicon glyphicon-search" aria-hidden="true"></span>

New:
{{glyphicon_search}}
```

**Is there anything you'd like reviewers to focus on?**
1. Wording of UserGuide.

**Testing instructions:**
1. Use `{{GLYPH_NAME}}` while authoring content and to see if  the Glyphicon appears on generated page from MarkBind.
